app/screens/navigation.ts

```diff
@@ -197,6 +197,7 @@ export function resetToHome(passProps: LaunchProps = {launchType: LaunchType.Nor
         dismissModal({componentId: Screens.LOGIN});
         dismissModal({componentId: Screens.SSO});
         dismissModal({componentId: Screens.BOTTOM_SHEET});
+        DeviceEventEmitter.emit(Events.FETCHING_POSTS, false);
```

Comment:
use case : 
You have server A and B... you are on B...then you remove B....the loader was not stopping.... this 'emit' fixes the issue.